tools/lkl/lib/virtio_net_macvtap.c

```diff
@@ -15,17 +15,26 @@
  * sudo chown thehajime /dev/tap22
  */
 
+#include <stdio.h>
 #include <net/if.h>
+#ifdef __linux__
 #include <linux/if_tun.h>
+#endif
 
 #include "virtio.h"
 #include "virtio_net_fd.h"
 
 struct lkl_netdev *lkl_netdev_macvtap_create(const char *path, int offload)
 {
+#ifdef __linux__
```

Comment:
I think it would be better to add LKL_HOST_CONFIG_VIRTIO_NET_MACVTAP and not set it for freebsd

Reply:
thanks, added the variable by checking `linux/if_tun.h`.